src/ObjectUtils/CoffFile.cpp

```diff
@@ -152,6 +154,22 @@ ErrorMessageOr<PdbDebugInfo> CoffFileImpl::GetDebugPdbInfo() const {
   return pdb_debug_info;
 }
 
+std::string CoffFileImpl::GetBuildId() const {
+  ErrorMessageOr<PdbDebugInfo> pdb_debug_info = GetDebugPdbInfo();
+  if (pdb_debug_info.has_error()) {
+    return "";
```

Comment:
LOG the error? (and the fact it is ignored)

Reply:
Done.